operations/patch_util.go

```diff
@@ -524,23 +528,74 @@ func validatePatchSize(diff *localDiff, allowLarge bool) error {
 	return nil
 }
 
+type outputPatchParams struct {
+	patches []patch.Patch
```

Comment:
I'm not a huge fan of having both patches and patch be options for these params. I think it can lead to bugs and has strange implications, like below where you have to assign "patch" for each call of getPatchDisplay to essentially point to what you want.

While I'm all for modularizing functions, getPatchesDisplay doesn't do _that_ much more than getPatchDisplay (actually it really just seems to do the same thing? It loops through the patches and then calls the same thing as 591-598 it looks like).

Can these functions just be combined into one getPatchesDisplay? A list of one patch can be passed in for the `evergreen patch` command.

Reply:
Feel free to reach out if you want to discuss this more

Reply:
That's fair. The only problem is that output json and outputing our generic output is a little different. When output json for a single or multiple, it should be either an array or a single object. While our generic output just has the same output repeated. I'll consolidate these two and make it have some checks I think